Makefile

```diff
@@ -75,6 +75,45 @@ dev: ## Build and install a development build
 	@cp $(GOPATH)/bin/packer bin/packer
 	@cp $(GOPATH)/bin/packer pkg/$(GOOS)_$(GOARCH)
 
+# Docker build variables and targets
+REGISTRY_NAME?=docker.io/hashicorp
+IMAGE_NAME=packer
+VERSION?=1.7.10
```

Comment:
can this version be parsed with `version.sh` so we don't have to modify it in two places?

Reply:
It's a great idea! The issue I ran into was while running some of the `make docker-*` commands using the version set in `version.go`. `make docker-official`, for ex, fails because the version is set to the next patch version which hasn't been released yet. I don't have a great solution right now, but we could think of something to benefit other projects as well.

Reply:
Going to resolve this for now. We can look into refactoring later on.